src/10_functions.py

```diff
@@ -1,6 +1,16 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
+def is_even(n):
+    if n % 2 == 0:
+        print("true")
+        return True
+    else:
+        print("false")
+        return False
```

Comment:
Excellent! One thing you may see others do is `return n%2 == 0` which will return a boolean itself. Either way is valid ofcourse.